mempool/src/core_mempool/transaction_store.rs

```diff
@@ -145,7 +158,12 @@ impl TransactionStore {
         sender: &AccountAddress,
         sequence_number: u64,
         is_rejected: bool,
+        metric_label: &str,
     ) {
+        if let Some(txn) = self.get_mempool_txn(sender, sequence_number) {
```

Comment:
why is this in different place from log_latency , in the remove_transaction above?

should we also have for "get_batch" this metric, it will probably be useful, and we can also see if different nodes are creating blocks with different fees as well.